### PR TITLE
Add updated sidenav

### PR DESCRIPTION
**Pull request by shreyasingh7673**
## Related Issue
  - sidenav project updation.
Closes: #[123]

### Describe the changes you've made
I have updated sidenav.html and added sidenav.css for changing current sidenav project into animated sidenav

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

### Checklist:

- [x ] My code follows the style guidelines of this project.
- [ x] I have performed a self-review of my own code.
- [ x] I have commented my code, particularly in hard-to-understand areas.
- [x ] I have made corresponding changes to the documentation.
- [x
Screenshot 2021-02-18 005011
 ] New and existing unit tests pass locally with my changes.

### Screenshots
Put any screenshot(s) of the project here.

### CODE REVIEW

**Comment by shreyasingh7673** [comment]
Please give me feedback in case this PR will not get merged because this is the first time I have participated in an open-source program so I can rectify my mistakes.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -1,17 +1,16 @@
 ---
```
Comment (Vishal-raj-1) [inline .github/ISSUE_TEMPLATE/feature_request.md:1]: @shreyasingh7673 Welcome to open source world !! You don't have to change in `feature_request.md` file. You have to edit it in Pull Request message.

Comment (shreyasingh7673) [reply .github/ISSUE_TEMPLATE/feature_request.md:1]: i got it. Do I need to send PR again?

Comment (shreyasingh7673) [reply .github/ISSUE_TEMPLATE/feature_request.md:1]: @Vishal-raj-1

**Comment by shreyasingh7673** [comment]
@Vishal-raj-1 resolved

**Comment by urvashi-code1255** [comment]
@shreyasingh7673 Your project is looking pretty cool.

Just do some things mentioned below:-
1. Edit your PR template.
   a) Write "Closes: #123" in place of "Closes: #123"
   b) Write [x] in place of [x ] there should be no space.
2. Also resolve the merge conflicts again.

**Comment by Vishal-raj-1** [comment]
@shreyasingh7673 Now you don't have to edit index.html. You have to add your project detail in script.js folder.

**Comment by urvashi-code1255** [comment]
Hey @shreyasingh7673 Any updates on this PR?

**Pull request merged**